config/static/nodes/rpm-overrides/Makefile

```diff
@@ -0,0 +1,26 @@
+POOL ?= worker master
+RPM_PACKAGES ?=
+# RPM_RUNC ?= https://ftweedal.fedorapeople.org/runc-1.0.0-990.rhaos4.8.gitcd80260.el8.x86_64.rpm
+# RPM_RUNC_PKG_NAME ?= runc-1.0.0-990.rhaos4.8.gitcd80260.el8
+
+.PHONY: configure
+configure:
+ifneq (,$(RPM_PACKAGES))
+	git checkout kustomization.yaml
```

Comment:
The same as before; do not checkout this file, and generate the head from here, and delete kustomize.yaml from repository and don't checkout it.